uPortal-api/uPortal-api-internal/src/main/java/org/apereo/portal/layout/dlm/remoting/GroupListHelperImpl.java

```diff
@@ -47,6 +47,9 @@
     /*
      * (non-Javadoc)
      * @see org.apereo.portal.layout.dlm.remoting.IGroupListHelper#search(java.lang.String, java.lang.String)
+     *
+     * External search, thus case insensitive.
```

Comment:
What do you mean by "external?"  (Searches based on user input are the ones that need to be insensitive... I imagine that's what you mean.)

Reply:
Yeah 'external' meant searches based on user input. I'll switch around that wording to be more clear.